instructions.md

```diff
@@ -0,0 +1,203 @@
+- Make `props` a function parameter
+- Use the `map()` method on the array prop
+- The `map()` callback function has a data object and an index as parameters
+- The callback should return a `Tile` component with the data object added to `props` and set the component `key` to the index 
+
+### Task #6 
+
+Based on the given requirements, implement `Tile` as a stateless component that renders the data from an object.
+
+This component will:
+- Have one prop: 
+  - An object
+- Use the `Object.values()` [method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values) to iteratively render the data in the object.
+
+### Hint
+
+- Make `props` a function parameter
+- `map()` over the array of values from the object in `props` using `Object.values()`
+- Use the `map()` callback to render each value in the array on a separate line
+
+### Task #7
+
+Based on the given requirements, implement `AppointmentsPage` as a stateful component that handles the logic for adding new appointments and listing current appointments.
+
+This component will:
+- Have three props:
+  - The current list of appointments
+  - The current list of contacts
+  - A function for adding a new appointment.
+- Render an `AppointmentForm` and a `TileList`.
+- Keep track of the current title, contact, date and time entered into the form.
+- Add a new appointment on form submission.
+- Clear the form on submission. 
+
+### Hint
+
+- Add `{ useState }` to the `'react'` import statement
+- Import `AppointmentForm` and `TileList`
+- Make `props` a function parameter
+- The appointment info state variables should default to an empty string using `useState('')`
+- The handle submit function should add contact function and set the contact info state variable to `''` 
+
+### Task #8
+
+Based on the given requirements, implement `AppointmentForm` as a stateless component that renders a web form to collect the necessary appointment information.
+
+This component will:
+- Have 9 props:
+  - The variables for appointment title, contact, date and time
+  - The setter functions for appointment title, contact, date and time
+  - The function that handles the form submission.
+- Render a `form` with:
+  - The `onSubmit` attribute set
+  - 3 controlled `input` components, one for each piece of appointment data
+  - a `ContactPicker` component with the contacts list as its props.
+  - A submit button
+- Use `getTodayString()` to set the minimum date of the data `input`.
+
+### Hint
+
+- Import the `ContactPicker` component
+- Make `props` a function parameter
+- Use `<form onSubmit={props.handler} >` for the form
+- The children of the form should be `<input>`s of `type` text, date, time and submit
+- The `value` attribute of the input fields should be set to the associated variable in `props`
+- The `onChange` attributes of the input fields should be set to a callback function that passes `e.target.value` to the the associated setter function
```

Comment:
See my comments for task #4 and make the language here match the suggested changes.